DebbeqGasha/DG001.xml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="DGDG001" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Sane Golgotā</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="GA"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas:  Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0965DGDG"/>
+                        <idno>DGDG-001</idno>
+                    </msIdentifier>
+
+                    <msContents>
+                        <summary/>
+
+                          <msItem xml:id="ms_i1">
+                              <locus from="4r" to="50v"/>
+                              <title type="complete" ref="LIT2246salotz"/>
+                                  <incipit xml:lang="gez">
+                              በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ጸሎት፡ ዘእግዝእትነ፡ ማርያም፡ ድንግል፡ ወላዲተ፡ አምላክ፡ እሙ፡ ለብርሃን፡ ጸሎታ፡ ወበረከታ፡
+                              የሃሉ፡ ምስለ፡ ገብራ፡ ወልደ፡ መድኅን፡ ለዓለመ፡ ዓለም፡ አሜን፨
+                                  </incipit>
+                                    <explicit xml:lang="gez">
+                                      ቡሩክ፡ አንተ፡ እግዚኦ፡ ይትአኮት፡ ስምከ፡ ወይትባረክ፡ ስመ፡ መንግሥትከ፡
+                                          ወምስለ፡ ቅዱስ፡ መንፈስ፡ ስብሐት፡ ወእኂዝ፡ ይእዜኒ፡ ወዘልፈኒ፡ ወለዓለመ፡ ዓለም፡ አሜን።
+
+                                  <msItem xml:id="ms_i2">
+                                      <locus from="51r" to="62v"/>
+                    <title type="complete" ref=""> </title>
```

Comment:
Now only an identification for this text is missing. I could not identify it with this incipit and explicit. If you could type some a longer portion for the incipit and explicit, this might help us. Do you know which text this is?

Reply:
it has no title but it is the prayer of abune Estnfase Kristos, which describes the crucifixion event on the body of our Lord Jesus Christ.

Reply:
Have you searched on BM whether this prayer already has a work record? I could not find it, but maybe you have more information. If you don't find it either, please create a work creation issue https://github.com/BetaMasaheft/Documentation/issues/new?assignees=&labels=works%2C+Text+Identification&template=new-work-creation.md&title=New+work+record+for, stating all information you have for this prayer, and ideally a longer incipit and explicit. Then we will be able to create a new work ID. Thank you for your patience!